### PR TITLE
Fixed `codeblock` error :wink:

### LINKED ISSUE #499: coafile.rst: Fix bad setting example codeblock markup
difficulty/newcomer
Opened by @mamat-rahmat at [Gitter](https://gitter.im/coala/coala?at=5a17699571ad3f87362236cb)

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #499 :+1:

#### Changes made   :- 
* Added `::` :blush:

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Users/coafile.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ This is the same file without section inheritance::
 All settings must be part of a section, so don't do this for implicit
 inheritance (this is also deprecated behavior). Implicit inheritance
 was leading to a section automatically getting inherited to all other
-sections without semantically making sense.
+sections without semantically making sense.::
```
Comment (khanchi97) [inline Users/coafile.rst]: @0x48piraj Remove trailing period, then it will look fine 👍

Comment (0x48piraj) [reply Users/coafile.rst]: But, everywhere the same pattern is followed.

Comment (khanchi97) [reply Users/coafile.rst]: Yeah, that's what I'm saying, everywhere the pattern should be same.But in your case, it should be like this
`sections without semantically making sense::`

Comment (0x48piraj) [reply Users/coafile.rst]: There's a full-stop at the last of sentence, so I left it. Should I remove the full-stop ? :blush:

Comment (khanchi97) [reply Users/coafile.rst]: yes, that's what I'm talking about.

Comment (prnvdixit) [reply Users/coafile.rst]: See for now, it looks like [this](https://screenshots.firefox.com/UvhCTZ76mDP3fb8f/github.com) 😃

**Comment by khanchi97** [comment]
You also have to squash your commits.If you don't know how to squash commits then follow this link http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

File: Users/coafile.rst (hunk 1 above)
Comment (khanchi97) [reply Users/coafile.rst]: yes, and it should look like this `making sense:`

**Comment by prnvdixit** [comment]
Also, fix your commit to adhere to https://coala.io/commit 😄

File: Users/coafile.rst (hunk 1 above)
Comment (0x48piraj) [reply Users/coafile.rst]: No @khanchi97 , `making sense:` reverts the whole issue back to zero :wink:

Comment (0x48piraj) [reply Users/coafile.rst]: And, `making sense::` does - `making sense:`
I think my first commit was OK. :blush:

**Comment by 0x48piraj** [comment]
**I am sorry, But I'm on Web Interface, I will take care from the next time ! :confused:**
A maintainer can merge and squash the commits easily.

> Reference : https://stackoverflow.com/questions/35948131/github-squash-commits-from-web-interface-on-pull-request-after-review-comments

File: Users/coafile.rst (hunk 1 above)
Comment (khanchi97) [reply Users/coafile.rst]: If you code like this `making sense::` then it will look like `making sense:` that you have done already.Now you only have to squash your commits 👍

Comment (0x48piraj) [reply Users/coafile.rst]: **I am sorry, But I'm on Web Interface, I will take care from the next time ! :confused:**
A maintainer can merge and squash the commits easily.

> Reference : https://stackoverflow.com/questions/35948131/github-squash-commits-from-web-interface-on-pull-request-after-review-comments

**Comment by mridubhatnagar** [comment]
Hi @0x48piraj 
Welcome to the community and thanks for the PR. And All the best for GCI-2017.

coala has certain guidelines mentioned for raising a Pull Request. Now, in order to get the PR merged
everyone here adhere's to those guidelines. These are as follows

1) Writing good commit messages. Writing a good commit message is a skill that everyone learns on the way of doing newcomer issues. For writing a good commit message we have http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html. 

So basically, this is how you are expected to do it. 

> filename in which you have made the changes:  write few words about what the change does

Describe in detail regarding the changes you have done. So, that it helps
the reviewer to review your Pull request and request or approve changes.

Closes  link to the issue your PR closes

**Comment by jayvdb** [comment]
@0x48piraj, please follow the instructions given to you by our developers.  We have good reasons for why and how we do things, which you will learn if you proceed through the newcomer process.
Our newcomer process exists only to ensure you know how to use `git` correctly.
If you dont follow our processes, we discard your PR.
Thank you.

**Comment by 0x48piraj** [comment]
Okay, I will do it all again :+1: 
But, I think you should have given me a chance, I said I will take care from next time.
I know how to use `git`, It's just that, the issue was so small that I used web-interface.
> If you dont follow our processes, we discard your PR.
I never said anything, It was just a request. :confounded:

**Comment by prnvdixit** [comment]
@0x48piraj 
> I think you should have given me a chance

You are still assigned to the issue - Nobody would or can take up a issue from you unless you are not working on it anymore. :tada:
As far as using web interface is concerned - We all (and hopefully you too) contribute to OSS to get familiar with ideal practices both for actual coding as well as code management. So, preferably you should respond firmly to any reviews (or critical comments) rather than "I never said anything" :wink:.
You might get away with using web interface for some of your issues but at the end of day, you __might__ have to move to ```git``` to solve some complex ones - So, why not use it from start itself 😛.
> I will do it all again

Also, it's a great first PR :smile: - you needn't start a new PR, just go through the suggested changes and associated guides 😃.

**Comment by 0x48piraj** [comment]
> Also, it's a great first PR :smile: - you needn't start a new PR, just go through the suggested changes and associated guides :smiley:.

I was talking about setting up the project in `git` :stuck_out_tongue: 
And "I never said anything" was for that this was just a request to accept this PR, and from next issue, I will do it via `git`.
Anyways, you all are right! I read the guidelines, but of my laziness I did it via web-interface.
I know I'm assigned :wink: 
Thanks @prnvdixit for such a positive comment.
From now, I will do as it is done in `coala` :blush:

**Comment by newbazz** [comment]
@0x48piraj are you still on this issue?

**Comment by 0x48piraj** [comment]
Actually No.
If they'll merge it as it is. Cool! Otherwise, You can work on this :+1:

**Comment by newbazz** [comment]
Sorry, but it cant be merged as it is.

**Comment by 0x48piraj** [comment]
:rofl: Okay @newbazz. Go on :rocket:
